neuro_san/internals/chat/connectivity_reporter.py

```diff
@@ -24,8 +24,7 @@
 
 class ConnectivityReporter:
     """
-    A class that knows how to report the connectivity of an entire
-    AgentToolRegistry.
+    A class that knows how to report the connectivity of an entire AgentNetwork.
```

Comment:
Fix comment. The refactoring to make this based on an AgentNetworkInspector has already been done.